distributedlog-core/src/main/java/org/apache/distributedlog/tools/DistributedLogTool.java

```diff
@@ -1665,7 +1665,7 @@ protected void repairLogSegment(BookKeeperAdmin bkAdmin,
                 System.out.println("Skip inprogress log segment " + segment);
                 return;
             }
-            LedgerHandle lh = bkAdmin.openLedger(segment.getLogSegmentId(), true);
```

Comment:
@leighst @zhaijack - I think this change is related to autorecovery chagnes in twitter branch. I haven't ported those changes. @zhaijack how about we create a github issue for this? and make a comment in the code to track this, once 4.6.0 is out we can fix this.